ajax/libs/elevator.js/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "elevator.js",
+  "version": "1.0.0",
+  "filename": "elevator.min.js",
+  "description": "Elevator.js fixes those awkward \"scroll to top\" moments the old fashioned way.",
+  "homepage": "http://tholman.com/elevator.js/",
+  "keywords": [
+    "scroll",
+    "top",
+    "elevator"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/tholman/elevator.js.git"
+  },
```

Comment:
I think we can follow the repository format as the same with upstream.
```js
  "repository": {
    "type": "git",
    "url": "git+https://github.com/tholman/elevator.js.git"
  },
```
ref: https://github.com/tholman/elevator.js/blob/master/package.json#L9-L12